test/integration/suites/upstream-authority-vault/conf/server/base/spire-server.yaml

```diff
@@ -0,0 +1,255 @@
+# ServiceAccount used by the SPIRE server.
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spire-server
+  namespace: spire
+
+---
+
+# Required cluster role to allow spire-server to query k8s API server
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: spire-server-cluster-role
+rules:
+  - apiGroups: [""]
+    resources: ["pods", "nodes"]
+    verbs: ["get", "list", "watch"]
+    # allow TokenReview requests (to verify service account tokens for PSAT
+    # attestation)
+  - apiGroups: ["authentication.k8s.io"]
+    resources: ["tokenreviews"]
+    verbs: ["get", "create"]
```

Comment:
this is not required on this suite, since you are not starting an agent and just mint SVIDs on demand

Reply:
[73858ac](https://github.com/spiffe/spire/pull/3977/commits/73858ac757c2d08981b7e93a253764e4aafc0eba)